### PR TITLE
[Community](readme)Update README.md

### PR DESCRIPTION
**Pull request by luzhijing**
# Proposed changes

add technical overview and fix some typo.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [x] Yes
    - [ ] No
    - [ ] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes
    - [x] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,37 +29,114 @@ under the License.
...
+
+As shown in the figure below, after various data integration and processing, the data sources are usually stored in the real-time data warehouse Apache Doris and the offline data lake or data warehouse (in Apache Hive, Apache Iceberg or Apache Hudi).
+
+<img src="https://doris.apache.org/assets/images/what-is-doris-2ed5ac7fffa3799871d5d33993b1de09.png">
+
+Apache Doris is widely used in the following scenarios:
+
+- Reporting Analysis
+
+	- Real-time Dashboards
```
Comment (morningman) [inline README.md:53]: use 4 spaces instead of tab

Comment (luzhijing) [reply README.md]: fixed

File: README.md [hunk 2]
```diff
@@ -68,87 +145,86 @@ Currently, the Apache Doris community has gathered more than 300 contributors fr
 
 We deeply appreciate 🔗[community contributors](https://github.com/apache/doris/graphs/contributors) for their dedication to Apache Doris.
```
Comment (jackwener) [inline README.md:146]:
```suggestion
We deeply appreciate 🔗[community contributors](https://github.com/apache/doris/graphs/contributors) for their contribution to Apache Doris.
```

**Review by morningman — APPROVED** [review]
LGTM

**Review by kpfly — APPROVED** [review]
LGTM

**Pull request merged**